qa/rolling-upgrade/src/test/resources/processor/PipelineForSparseEncodingProcessorConfiguration.json

```diff
@@ -3,7 +3,7 @@
   "processors": [
     {
       "sparse_encoding": {
-        "model_id": "%s",
+        "model_id": "%s",{{batch_size}}
```

Comment:
Can we use the same style of filling the model_id?

Reply:
this is an optional parameter, sometimes, we just don't use it, so we can replace it with empty string

Reply:
For this case can we set it as 1 (the default value)?

Reply:
That also works